.github/workflows/lint.yml

```
@@ -0,0 +1,20 @@
+name: Lint
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+jobs:
+  lint:
+    name: Lint Files
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+      - uses: actions/setup-node@v2
```

```suggestion
        uses: actions/setup-node@v2
```
If you put a `-` it will be another task if i recall correct.